pkg/gofr/config/godotenv_test.go

```diff
@@ -13,36 +14,72 @@ func TestNewEnvFile(t *testing.T) {
 }
 
 func TestNewGoDotEnvProvider(t *testing.T) {
-	f := new(EnvFile)
+	var (
+		configs = "TEST=test\nNAME=gofr\nKEY_123=value123"
```

Comment:
same can be tested here about special characters and different possible keys and values